sql/schema_changer_test.go

```diff
@@ -510,6 +510,7 @@ func runSchemaChangeWithOperations(
 // that run simultaneously.
 func TestRaceWithBackfill(t *testing.T) {
 	defer leaktest.AfterTest(t)()
+	t.Skip("#7628")
```

Comment:
Yes but having names in code improves accountability.

On Thu, Jul 7, 2016 at 11:52 AM, Tobias Schottdorf <notifications@github.com
> wrote:

> In sql/schema_changer_test.go
> <https://github.com/cockroachdb/cockroach/pull/7675#discussion_r69933239>:
>
> > @@ -510,6 +510,7 @@ func runSchemaChangeWithOperations(
> >  // that run simultaneously.
> >  func TestRaceWithBackfill(t *testing.T) {
> >  	defer leaktest.AfterTest(t)()
> > +	t.Skip("#7628")
>
> Will do if the CI eats it. Otherwise I'm not convinced that the assignee
> needs to be here (as the assignee of an issue can freely change and in fact
> should for test failures based on who's available).
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/cockroachdb/cockroach/pull/7675/files/74cc6901b7f5026fc819b42ec35468b1f55c8e82#r69933239>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABdsPHMaklJ8SXdrNgqfMXPdSdvCoc5Lks5qTSDLgaJpZM4JHM3V>
> .
>